ui/helpers/utils/util.js

```diff
@@ -50,6 +50,12 @@ export function isDefaultMetaMaskChain(chainId) {
   return false;
 }
 
+// Both inputs should be strings. This method is currently used to compare tokenAddress hex strings.
+export function isEqualCaseInsensitive(value1, value2) {
+  if (typeof value1 !== 'string' || typeof value2 !== 'string') return false;
+  return value1?.toLowerCase() === value2?.toLowerCase();
```

Comment:
Nit: are the `?`'s necessary? given both `value1` and `value2` will be of type `string` at this comparison

Reply:
no they are not required anymore. Will remove!